digitalpublicgoods/synthea.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Synthea",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/synthetichealth/synthea#license"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
```

Comment:
My cursory view seems to indicate no, but since Java is sort of @iperdomo's thing, I'll defer to his expertise for a closer look.